flycheck.el

```diff
@@ -5087,7 +5110,7 @@ tool, just like `compile' (\\[compile])."
 
 
 ;;; General error parsing for command checkers
-(defun flycheck-parse-output (output checker buffer)
+(defun flycheck-parse-output (output checker buffer) 
```

Comment:
Please be careful to not introduce spurious whitespace changes.  Run `M-x delete-trailing-whitespace` on this buffer after editing, and take a look at `whitespace-cleanup-mode` or `ws-butler` to automate this process.